client/src/StudentHomePage/StudentHomePage.scss

```diff
@@ -0,0 +1,26 @@
+.Home{
+    text-align: left;
+    margin-left: 5%;
+    margin-right: 5%;
+    margin-top: 5%;
+    margin-bottom: 5%;
+}
+th, td {
+    padding: 25px;
```

Comment:
Can you use measures that are dynamic to screen size? Changing px to either % or em would do.